cli/drop/icon_drop_surface_ocean.sh

```diff
@@ -24,8 +24,8 @@ exp="historical-1990"
 for variable in avg_sos avg_tos ; do
```

Comment:
revert all the changes in this file. This was a script specifically to fix ICON data, which in principle is completely deprecated, but will always need to point to LRA, not to DROP